### PR TITLE
docs(tutorial): add k8s fluentbit observability tutorial

### PR DESCRIPTION
**Pull request by antoinechampion**
### Your checklist for this pull request

- [X] Check that the commit messages match our requested structure.
- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Tutorial for sending Kapsule logs and metrics to the Observability Cockpit using Fluent Bit. See https://github.com/scaleway/docs-content/issues/1517.

### CODE REVIEW

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+### Exploring metrics
+
+Grafana has a built-in dashboard for visualizing node metrics.
+1. Go to **Dashboards** in your Grafana instance
+2. Click on **New**, **Folder** and name it `Kapsule`
+3. Click on **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
+    ```
+    https://grafana.com/grafana/dashboards/405-node-exporter-server-metrics/
+    ```
+4. After clicking **Load** you can access the new dashboard named **Node Exporter Server Metrics**.
```
Comment (RoRoJ) [inline tutorials/k8s-fluentbit-observability/index.mdx:226]:
```suggestion
4. Click **Load** to access the new dashboard named **Node Exporter Server Metrics**.
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+## Visualizing Kapsule logs and metrics
+
+Logs and metrics from your Kubernetes cluster can now be found in your Cockpit's dashboard
+
+### Exploring metrics
+
+Grafana has a built-in dashboard for visualizing node metrics.
+1. Go to **Dashboards** in your Grafana instance
+2. Click on **New**, **Folder** and name it `Kapsule`
```
Comment (RoRoJ) [inline tutorials/k8s-fluentbit-observability/index.mdx:221]:
```suggestion
2. Click **New**, **Folder** and name it `Kapsule`
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+3. Click on **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
```
Comment (RoRoJ) [inline tutorials/k8s-fluentbit-observability/index.mdx:222]:
```suggestion
3. Click **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
```

**Review by RoRoJ — COMMENTED** [review]
Thanks for your tutorial :-) I have left some comments, mostly because we prefer to use numbered steps where possible (one step = one action, and should start with an action verb).

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+The last step in the Fluent Bit configuration is to define where the logs and metrics will be pushed.
+
+First, generate a Cockpit token:
+1. Go to your Cockpit in the **API and Tokens tab**.
+2. Navigate to the **Token section** and click on **Generate a token**.
+3. Choose both Push permissions for metrics and logs.
+
+<Lightbox src="scaleway-cockpit-token-permissions.webp" alt="" />
+
+In the `values.yaml` file, create a section named `config.outputs`:
```
Comment (nerda-codes) [inline tutorials/k8s-fluentbit-observability/index.mdx:166]:
```suggestion
2. Create a section named `config.outputs` in the `values.yaml` file:
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+2. `Header`: Next to `Bearer`, paste the token generated in the previous step.
+3. `Host` from the second subsection: paste your Logs API URL defined in the **API and Tokens tab** section from the Cockpit. Remove the `https://` protocol.
+4. `Http_passwd`: paste the token generated in the previous step.
+
+In the first subsection, the `prometheus_remote_write` plugin is used to send metrics to the [Prometheus](https://prometheus.io/) server of your Cockpit using the remote write protocol.
+In the second subsection, the `loki` plugin is used to send logs to the [Loki](https://grafana.com/oss/loki/) server of your Cockpit, using the field mapping from `labelmap.json` defined above.
+
+## Installing Fluent Bit
+
+In the same directory as your `values.yaml` file, run:
```
Comment (nerda-codes) [inline tutorials/k8s-fluentbit-observability/index.mdx:205]:
```suggestion
Run the following command in the same directory as your `values.yaml` file:
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+We will configure Fluent Bit to retrieve the metrics (CPU, memory, disk usage...) from your Kubernetes nodes and the applicative logs from your running pods.
+
+In the `values.yaml` file, create a new section `config.inputs`:
+
+```yaml
+  inputs: |
+    [INPUT]
+        Name                  node_exporter_metrics
+        Tag                   node_metrics
+        Scrape_interval       2
```
Comment (blazarecki-scaleway) [inline tutorials/k8s-fluentbit-observability/index.mdx:82]: Can you update the scrape_interval to be `60s` ? 
A `2s` scrape interval will increase the cost a lot on Cockpit.
Having it to `60s` divide the price by 30.

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+Logs and metrics from your Kubernetes cluster can now be found in your Cockpit's dashboard
+
+### Exploring metrics
+
+Grafana has a built-in dashboard for visualizing node metrics.
+1. Go to **Dashboards** in your Grafana instance
+2. Click **New**, **Folder** and name it `Kapsule`
+3. Click **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
+    ```
+    https://grafana.com/grafana/dashboards/405-node-exporter-server-metrics/
```
Comment (blazarecki-scaleway) [inline tutorials/k8s-fluentbit-observability/index.mdx:221]: I think this dashboard `https://grafana.com/grafana/dashboards/1860-node-exporter-full/` is better, what do you think ? The one you provided is from 2018 and this one is up to date. (And nicer but it's a matter of taste)

**Review by blazarecki-scaleway — CHANGES_REQUESTED** [review]

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 8]
```diff
@@ -0,0 +1,230 @@
...
+We will configure Fluent Bit to retrieve the metrics (CPU, memory, disk usage...) from your Kubernetes nodes and the applicative logs from your running pods.
+
+In the `values.yaml` file, create a new section `config.inputs`:
+
+```yaml
+  inputs: |
+    [INPUT]
+        Name                  node_exporter_metrics
+        Tag                   node_metrics
+        Scrape_interval       2
```
Comment (nerda-codes) [inline tutorials/k8s-fluentbit-observability/index.mdx:82]:
```suggestion
        Scrape_interval       60
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 9]
```diff
@@ -0,0 +1,230 @@
...
+Logs and metrics from your Kubernetes cluster can now be found in your Cockpit's dashboard
+
+### Exploring metrics
+
+Grafana has a built-in dashboard for visualizing node metrics.
+1. Go to **Dashboards** in your Grafana instance
+2. Click **New**, **Folder** and name it `Kapsule`
+3. Click **New**, **Import** and paste the following URL in the **Import via grafana.com** field:
+    ```
+    https://grafana.com/grafana/dashboards/405-node-exporter-server-metrics/
```
Comment (nerda-codes) [inline tutorials/k8s-fluentbit-observability/index.mdx:221]:
```suggestion
    https://grafana.com/grafana/dashboards/1860-node-exporter-full/
```

File: tutorials/k8s-fluentbit-observability/index.mdx [hunk 10]
```diff
@@ -0,0 +1,230 @@
...
+
+In the first subsection, the `prometheus_remote_write` plugin is used to send metrics to the [Prometheus](https://prometheus.io/) server of your Cockpit using the remote write protocol.
+In the second subsection, the `loki` plugin is used to send logs to the [Loki](https://grafana.com/oss/loki/) server of your Cockpit, using the field mapping from `labelmap.json` defined above.
+
+## Installing Fluent Bit
+
+Run the following command in the same directory as your `values.yaml` file:
+
+```
+helm upgrade --install fluent-bit fluent/fluent-bit -f .\values.yaml
```
Comment (nerda-codes) [inline tutorials/k8s-fluentbit-observability/index.mdx:205]:
```suggestion
helm upgrade --install fluent-bit fluent/fluent-bit -f ./values.yaml
```

**Review by nerda-codes — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Comment by antoinechampion** [comment]
Apologies for the late reply I was away for a bit. Thank you for taking care of the fit and finish!

**Pull request merged**